packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -574,11 +579,13 @@ class Auth {
             responseValue,
             defaultValue
         )
+        const {uido} = this.parseSlasJWT(res.access_token)
         const expiresDate = this.convertSecondsToDate(refreshTokenTTLValue)
         this.set('refresh_token_expires_in', refreshTokenTTLValue.toString())
         this.set(refreshTokenKey, res.refresh_token, {
             expires: expiresDate
         })
+        this.set('uido', uido)
```

Comment:
We are setting this value.. but where are we getting it?

Reply:
We are getting it here `const {uido} = this.parseSlasJWT(res.access_token)` in line 582